src/AsyncWebSocket.cpp

```diff
@@ -840,7 +840,7 @@ void AsyncWebSocketClient::binary(AsyncWebSocketMessageBuffer * buffer)
 
 IPAddress AsyncWebSocketClient::remoteIP() {
     if(!_client) {
-        return IPAddress(0U);
+        return IPAddress((uint32_t) 0);
```

Comment:
Instead of using a type cast I'd just use a more specific type suffix:
```suggestion
       return IPAddress(0UL);
```

Reply:
Actually, using `0UL` breaks ESP32. I think the only proper way to fix this is using a static type cast.

Reply:
Can we return IPADDR_ANY or IPADDR_NONE here? IPADDR_ANY looks good for this, no?